README.md

```diff
@@ -25,13 +25,24 @@ The following datasets and model architectures are implemented.
 
 ### Tagging / Recognition
 
-The goal of tagging is to infer a set of labels for each image. The following datasets and model architectures are implemented.
+The goal of tagging is to infer a set of labels for each image. Here are some examples of training chips from the Planet Kaggle competition with labels predicted by our best single-model network, a DenseNet121 using an SGD optimizer.
```

Comment:
I think we should put the link to the competition in here too. Also, I thought we used Adam for our best DenseNet121 experiment. We should link to the experiment file for the best run too.